### PR TITLE
Capybara rspecs

### PR DESCRIPTION
**Pull request by Aeropio**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -20,9 +20,8 @@ class User < ApplicationRecord
   has_many :owned_groups, class_name:'Group', foreign_key: :owner_id, dependent: :destroy
   has_many :group_members, dependent: :destroy
   has_many :groups, through: :group_members
-  has_many :projects, as: :projectable, dependent: :destroy
   has_many :project_members, dependent: :destroy
-  has_many :projects, through: :project_members
+  has_many :projects, through: :project_members, as: :projectable, dependent: :destroy
```
Comment (chandra-seanergy) [inline app/models/user.rb]: is this working?????

File: spec/features/5_project_members_functionality_spec.rb [hunk 2]
```diff
@@ -42,4 +42,40 @@ def create_projects
       expect(page).to have_selector 'span', class: 'member-name', text: user.name
     end
   end
+
+  # scenario "User Search should search all users matching the entered phrase" do
```
Comment (chandra-seanergy) [inline spec/features/5_project_members_functionality_spec.rb]: don't commit the comment line

File: app/controllers/projects_controller.rb [hunk 3]
```diff
@@ -29,11 +29,11 @@ def show
 
     # Delete project from database
     def delete_project
-      if @project.destroy
-        render json: {status: 200, message: "Project deleted successfully.", project: @project}
-      else
-        render json: {status: 500, message: @project.errors.full_messages}
-      end
+      # if @project.destroy
+      #   render json: {status: 200, message: "Project deleted successfully.", project: @project.attributes.merge(avatar: @project.project_avatar)}
```
Comment (DhruvRSeanergy) [inline app/controllers/projects_controller.rb]: Why is delete project commented?

File: spec/features/4_project_functionality_spec.rb [hunk 4]
```diff
@@ -89,14 +90,8 @@ def create_projects
     end
     click_button 'Update Project'
     expect(page).to have_selector 'div', class: 'user-sucess', text: 'Project updated successfully.'
-    find('div.group-menu-btn').click
-    find('div.group-menu-items', text: "Your projects").click
-    expect(page).not_to have_selector 'span', class: 'ellipsis-span', text: 'Test Public Project'
-    expect(page).to have_selector 'span', class: 'ellipsis-span', text: 'Test Create Public Project changed to Private'
```
Comment (DhruvRSeanergy) [inline spec/features/4_project_functionality_spec.rb]: Why is the complete scenario removed from specs

File: app/controllers/projects_controller.rb (hunk 3 above)
Comment (Aeropio) [reply app/controllers/projects_controller.rb]: oh it was giving stack too deep error . i will check it out

**Pull request merged**